README.md

```diff
@@ -40,13 +40,14 @@ SELECT * WHERE {
 
 By default, it generates CSV output in a form similar to:
 ```csv
-name;id;error;errorDescription;failures;hash;httpRequests;httpRequestsMax;httpRequestsMin;replication;results;resultsMax;resultsMin;time;timeMax;timeMin;timestamps;timestampsMax;timestampsMin
-C1;0;false;;0;d632b8166f912f4afd062d64186f2dc6;1766.5;2271;1262;2;6;6;6;1364;1398;1330;1363.5 1363.5 1363.5 1364 1364 1364;1398 1398 1398 1398 1398 1398;1329 1329 1329 1330 1330 1330
-C1;1;false;;0;e00f199d535cd1710bf9be67f04f39e4;1803.5;2308;1299;2;4;4;4;212;214;210;211.5 212 212 212;213 214 214 214;210 210 210 210
-C1;2;false;;0;c4499554f796e968a069e67a8f5d9d1c;1834.5;2339;1330;2;1;1;1;175.5;176;175;175.5;176;175
-C1;3;false;;0;6c0a9fe8be642ee232c10c9996912b97;2279.5;2784;1775;2;14;14;14;1747;1796;1698;1746 1746 1746 1746 1746 1746 1746.5 1746.5 1746.5 1746.5 1747 1747 1747 1747;1795 1795 1795 1795 1795 1795 1795 1795 1795 1795 1796 1796 1796 1796;1697 1697 1697 1697 1697 1697 1698 1698 1698 1698 1698 1698 1698 1698
-C1;4;false;;0;7536d3a2c1abc2a9ac92b1860efa3282;2522.5;3027;2018;2;8;8;8;1360;1373;1347;1355.5 1355.5 1355.5 1355.5 1359 1359 1359 1359;1372 1372 1372 1372 1372 1372 1372 1372;1339 1339 1339 1339 1346 1346 1346 1346
+name;id;error;errorDescription;failures;hash;replication;results;resultsMax;resultsMin;time;timeAggregate;timeMax;timeMin;timestamps;timestampsMax;timestampsMin;timestampsStd;timeStd
```

Comment:
Just so I understand correctly, the additions of this PR are the columns `timeAggregate`, `timestampsStd`, and `timeStd`?
If yes, then I'd be fine with these columns just always being emitted. Then we don't need a flag.

But what does `timeAggregate` mean exactly? Is it a sum?

Reply:
`timeAggregate` is all the execution time of the queries e.g: `18 17 17 16 18` for an average of `17.2` a min of `16` and a max of `17`. The PR also add the std for the number of HTTP request.

Reply:
Maybe a flag is not useful at all, but I think someone told me to put one, but I think the timestamps take for more memory and space then the time of each iteration.

Reply:
> timeAggregate is all the execution time of the queries e.g: 18 17 17 16 18

Ok, so it's for having all execution times instead of only the average.
Perhaps we can rename it to `times`?
Since it's not really an aggregate.

And let's remove the flag then indeed, to simplify things :-)